get_goes_x.py

```diff
@@ -18,17 +18,20 @@
 
 url = 'ftp://ftp.swpc.noaa.gov/pub/lists/xray/Gp_xr_5m.txt'
 
+last_err = None
```

Comment:
Is just adding a var outside the try/loop acceptable?  I'm not sure if this code worked in Py2, but in Py3 without some change if the tries fail I just get an error that "err" is undefined when the print statement tried to do its thing.